.github/workflows/cmake.yml

```diff
@@ -148,17 +148,14 @@ jobs:
         run_worked=y
         ninja run || run_worked=n
         echo "::endgroup::"
-        echo "::group::cat debug.log # Serenity output"
-        if [ "y" = "${run_worked}" ] ; then
-          cat debug.log
-        else
-          echo "skipped (qemu had non-zero exit-code)"
-        fi
-        echo "::endgroup::"
         [ "y" = ${run_worked} ]
```

Comment:
The run_worked thing doesn't seem to be used anymore... now that dumping the log is a second step?

Reply:
Ah true. I can clean this up more. The previous implementation BenW put here was clever, but if the ninja run step timed out it wouldn't dump the kernel log... which is kinda the opposite of what we want. That's when the debug log is most useful.